hs_discover/public/index.html

```diff
@@ -1,17 +1,218 @@
+                                        </div>
+
+                                        <div class="col-sm-6">
+                                            <img src="{{ STATIC_URL }}img/resource-icons/modelinstance48x48.png"
+                                                 alt="Model Instance Resource Icon" class="table-res-type-icon"/>
+                                            <small>&nbsp;Model Instance</small>
+                                        </div>
+
+                                        <div class="col-sm-6">
+                                            <img src="{{ STATIC_URL }}img/resource-icons/modflow48x48.png"
+                                                 alt="Collection Resource Icon" class="table-res-type-icon"/>
+                                            <small>&nbsp;MODFLOW Model Instance</small>
+                                        </div>
+
+                                        <div class="col-sm-6">
+                                            <img src="{{ STATIC_URL }}img/resource-icons/swat48x48.png"
+                                                 alt="SWAT Model Instance Resource Icon" class="table-res-type-icon"/>
+                                            <small>&nbsp;SWAT Model Instance</small>
+                                        </div>
+
+                                        <div class="col-sm-6">
+                                            <img src="{{ STATIC_URL }}img/resource-icons/webapp48x48.png"
+                                                 alt="Web App Resource Icon" class="table-res-type-icon"/>
+                                            <small>&nbsp;Web App</small>
+                                        </div>
+
+                                        <div class="col-sm-6">
+                                            <img src="{{ STATIC_URL }}img/resource-icons/modelprogram48x48.png"
+                                                 alt="Model Program Resource Icon" class="table-res-type-icon"/>
+                                            <small>&nbsp;Model Program</small>
+                                        </div>
+
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
-    <script src=/static/js/chunk-vendors.js></script>
-    <script src=/static/js/app.js></script>
+</div>
+<script src=/static/js/chunk-vendors.js></script>
+<script src=/static/js/app.js></script>
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+<script src=/static/js/map.js></script>
+<script src="https://unpkg.com/@google/markerclustererplus@4.0.1/dist/markerclustererplus.min.js"></script>
+<script async defer
+  src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBv1tggGSW3-12h6vkbo8BL711KaUnG1w0&callback=initMap&libraries=&v=weekly&libraries=places"
+></script>
```

Comment:
this is the prod google maps key how is this handled for deployments @dan-palmer1 @martinseul

Reply:
dev4, dev6, dev7 may want fully prod google maps keys for testing/stakeholders/demos, anyway.

Reply:
Did you check our production maps key into source control?  Why do this?  We had successfully kept our key out of source control before this commit.  It may not be a large security concern (I don't feel like I can speak to this without much research, but the fact that it's rendered in a web page suggests it may not be a security issue, but I don't know).  Regardless, previous developers felt like this should be kept out of source control and I don't think you should be checking it in without discussion with the rest of the team.  @martinseul - you manage the google account for this key, maybe you can chime in.

Reply:
as a general rule it should be kept out of source control so use of google maps on non prod systems does not count against our quota.

Reply:
Discussed google keys are public and whitlisted IPs should guard usage. Removed from this html file as a best practice, though. 6ce5346f10a3a862918aeb20679ff173e26d9f0b

Reply:
I don't see this commit.  The PR commit history shows the latest commit was on Sep 25.  Maybe you need to push?